openstates/cli/update.py

```diff
@@ -364,6 +397,14 @@ def _tb_info(type, value, tb):  # type: ignore
         sys.excepthook = _tb_info
 
     logging.info(f"Module: {args.module}")
+
+    # delete all objects from S3 folder for current module
+    try:
```

Comment:
There's no protection here for non-realtime runs. Right now, _all_ modules that execute `update.py` will try to delete objects from S3. And likely fail. We need to only perform this step when working with realtime data.

Reply:
Thanks for the catch, updating it